pixel/body.html

```diff
@@ -63,11 +63,10 @@
         '{{settings.organizationId}}',
         '{{settings.accountName}}',
         {
-          baseLiveAgentContentURL:
-            'https://c.la1-c2cs-ia2.salesforceliveagent.com/content',
+          baseLiveAgentContentURL: window.decodeURIComponent('{{settings.baseLiveAgentContentURL}}'),
```

Comment:
I think we should have the previous values as default values, otherwise other stores that might use this will break.

Reply:
Fixed